java/src/Ice/src/main/java/com/zeroc/Ice/FileLoggerI.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.Ice;
+
+final class FileLoggerI extends LoggerI {
+    private final java.io.FileOutputStream _out;
+
+    FileLoggerI(String prefix, String file) {
+        super(prefix);
+
+        if (file.isEmpty()) {
+            throw new InitializationException("FileLogger: file name is empty");
+        }
+
+        try {
+            _out = new java.io.FileOutputStream(new java.io.File(file), true);
+        } catch (java.io.FileNotFoundException ex) {
+            throw new InitializationException("FileLogger: cannot open " + file);
```

Comment:
Do we need to throw our own exception here? 

If so, the error message should include that the file is not found. We should also watch `SecurityException` in case the user does not have permission.

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/FileOutputStream.html#%3Cinit%3E(java.io.File,boolean)

Reply:
This was copied from the old code.

Reply:
We should open an issue to fix this.